node/Cargo.toml

```diff
@@ -99,7 +100,15 @@ manta-primitives = { path = '../runtime/primitives' }
 substrate-build-script-utils = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
 
 [features]
-default = []
+default = ["cli"]
+cli = [
+	"try-runtime-cli",
+]
+try-runtime = [ 
+	"manta-pc-runtime/try-runtime",
+	"calamari-runtime/try-runtime",
+]
+
```

Comment:
Not quite sure what this is for. Check the error again when it is commented out.